content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -219,30 +201,35 @@ Batch billing
 
 Vendor bills can be processed and managed in batches in the *Accounting* app.
 
-To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
-:guilabel:`checkbox` at the top left of the page, beside the :guilabel:`Number` column, under the
-:guilabel:`Create` button. This selects all existing vendor bills with a :guilabel:`Posted` or
-:guilabel:`Draft` :guilabel:`Status`.
+Navigate to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
+:guilabel:`checkbox` in the top-left corner, beside the :guilabel:`Number` column, under the
+:guilabel:`New` button.
```

Comment:
```suggestion
Navigate to :menuselection:`Accounting app --> Vendors --> Bills`. Then, tick the
checkbox in the top-left corner, beside the :guilabel:`Number` column, under the
:guilabel:`New` button.
```

Reply:
This is from an older version of the doc.